server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -78,6 +71,27 @@ export function create(
 		throttleIdSuffix: Constants.alfredRestThrottleIdSuffix,
 	};
 
+	// Todo: delete it and use the function in throttlerMiddleware.ts
+	const tenantThrottle = (throttleApi: string) => {
```

Comment:
R11s-base can use throttlerMiddleware from services-utills right now. Historian is the only place you need to duplicate this.

Reply:
Thanks. Resolved. :)